esp/xslt/esdl2monitor.xslt

```diff
@@ -203,32 +204,47 @@ END;
     string100 Message {xpath('Message')};
   END;
 
-  soapoutRec := record
+  soapoutRec := record <xsl:choose><xsl:when test="$platform='ESP'">(the_responseLayout)
```

Comment:
Minor, but can we use the "pretty" xml format? elements in new lines, and indented